felix/bpf/ut/attach_test.go

```diff
@@ -237,7 +238,7 @@ func runAttachTest(t *testing.T, ipv6Enabled bool) {
 		Expect(err).NotTo(HaveOccurred())
 		hasXDP := false
 		for _, p := range progs {
-			if p.Name == "cali_xdp_preamb" {
+			if strings.Contains(p.Name, "cali_xdp_preamb") {
```

Comment:
what is the new format returned? Might be worth adding such info in commit message because (a) is not reviewable and (b) next time anyone looks at that, there is no context.